HW4 (2)/index.js

```diff
@@ -0,0 +1,44 @@
+const data = localStorage.getItem("data") || localStorage.getItem("data") === {} ? 
+JSON.parse(localStorage.getItem("data")) :
+{
+    timeStamp: "",
+    turnedOn: true,
+};
+
+let switcher = document.querySelector(".switcher");
+let body = document.querySelector("body");
+let timeText = document.querySelector(".time");
```

Comment:
```suggestion
const switcher = document.querySelector(".switcher");
const body = document.querySelector("body");
const timeText = document.querySelector(".time");
```

Все що не змінюємо - кладемо в константи